custom_messages/msg/FollowerCommand.msg

```diff
@@ -0,0 +1,7 @@
+string  commandType
```

Comment:
Instead of Strings these should be based on integers with defined constant values. This way there is less coordination required between nodes, and the message definition itself contains a lot of the information needed.

Example is the `radiation_type` here http://docs.ros.org/melodic/api/sensor_msgs/html/msg/Range.html